packages/react-native/Libraries/AppDelegate/RCTAppDelegate.mm

```diff
@@ -144,10 +146,16 @@ - (RCTBridge *)createBridgeWithDelegate:(id<RCTBridgeDelegate>)delegate launchOp
   return [[RCTBridge alloc] initWithDelegate:delegate launchOptions:launchOptions];
 }
 
+- (void)customizeRootView:(RCTRootView *)rootView
+{
+  // Override point for customization after application launch.
```

Comment:
Shouldn't the default implementation set the color to `systemBackground`?

Reply:
nope, because otherwise we will overwrite the settings that apps might have setup in the `createRootViewWithBridge:` method.

The flow is currently:
1. `applicationDidFinishLaunching:` calls `createRootViewWithBridge:`
2. the `createRootViewWithBridge:` sets the backgroundColor to `systemBackground`
3. subclass might change the background color to something else
4. we call customiseView.

If customiseView sets back the color to `systemBackground`, we might lose the changes in step 3, where the users get warned.